queries.sql

```diff
@@ -7,4 +7,28 @@ SELECT date_of_birth from animals WHERE name = 'Pikachu' OR name = 'Agumon';
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
```

Comment:
- [ ] You should update the animals table by setting the species column to unspecified, then roll back the change. Means you should verify that change was made and rollbacked. Kindly, make sure that you included all example as in this example bellow:
```
 BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals;
```